_posts/en/newsletters/2021-01-13-newsletter.md

```diff
@@ -98,17 +98,17 @@ question below to see a summary of the answer from the meeting.*
 [Add unit testing of node eviction logic][review club
 #20477] is a PR ([#20477][Bitcoin Core #20477]) by
 pseudonymous contributor practicalswift to improve test coverage of Bitcoin Core's peer
-eviction logic when a node's inbound connection slots are full. Extreme care
+eviction logic when a node's inbound connection slots are full. Care
 must be taken to avoid exposing the node to attacker-triggered network
-partitioning with this logic, so this code is critical.
+partitioning with this logic.
 
 Most of the discussion focused on understanding Bitcoin Core's peer eviction
 logic.
 
 {% include functions/details-list.md
   q0="Inbound or outbound peer selection: which one is our primary defense
       against eclipse attacks?"
-  a0="Outbound peer selection, as outbound peers are far less under attacker
+  a0="Outbound peer selection, as outbound peers are far less likely to be under attacker
```

Comment:
The changes in git range-diff e7eb3712 de0f38d bb9f28e LGTM except maybe this one, as it changes the meaning; my thinking was that "outbound peers are far less under attacker control" because we choose them.